products/prisma-cloud/docs/cspm/AWS-cloud-account-onboarding.md

```diff
@@ -0,0 +1,550 @@
+---
+id: aws-cloud-account-onboarding
+title: AWS Cloud Account Onboarding on to Prisma Cloud using Automation
```

Comment:
@sridharam change the title to something more actionable like "Onboard AWS Cloud Account to Prisma Cloud..."